content/root/glossary.textile

```diff
@@ -0,0 +1,212 @@
+
+See the "capabilities documentation":/core-features/authentication/#capabilities-explained for further information
+
+h2(#channels). Channels
+
+Channels are used to logically separate messages into different topics. Clients subscribe to channels to receive the messages sent on them. When a publisher sends a message on a channel, all subscribers receive that message.
+
+See the "channels documentation":/core-features/channels for further information
+
+h2(#channel-namespaces). Channel namespaces
+
+Channel namespaces provide a way for certain properties (such as persistence) to be assigned to a group of channels, rather than requiring them to be assigned to individual channels. The namespace is the first colon-delimited segment of a channel name, or the channel name itself if no colon exists. For example, the channels @finance@ and @finance:approvals@ would both be considered part of a @finance@ namespace.
+
+See the "channel namespace documentation":/realtime/channels#channel-namespaces for further information.
+
+h2(#connection). Connection
+
+A connection to Ably is must be established and maintained for realtime communication. A connection has several connection states throughout the lifecycle of a connection, such as @connecting@, @connected@ and @suspended@.
+
+See the "connection documentation":/realtime/connection for further information.
+
+h2(#dashboard). Dashboard
+
+The Ably dashboard is the user interface for managing applications, viewing statistics, creating API keys and setting up integrations and channel rules.
+
+Access your Ably dashboard by "logging in":https://ably.com/login or "signing up":https://ably.com/sign-up.     
+
+h2(#dead-letter). Dead letter queue
+
+Dead letter queues store messages that were intended to be delivered to a queue, but have failed to be processed correctly. They are created when a queue is provisioned, with one dead letter queue per application, not per queue.
+
+See the "dead letter queue documentation":/general/queues#deadletter for further information.
+
+h2(#deltas). Deltas
+
+Deltas (or delta compression) are a feature that enable a subscriber to only receive the difference between the payload of a message and its predecessor. They are used when there is a degree of similarity between successive messages.
+
+See the "deltas documentation":/realtime/channels/channel-parameters/deltas for further information.
+
+h2(#firehose). Firehose
+
+Reactor Firehose is a feature that allows the realtime data published in Ably to be streamed directly into another streaming or queuing service such as Amazon Kinesis or Kafka.
```

Comment:
Should we link to Amazon's Kinesis and Kafka pages here?

Reply:
Thanks for pointing this out. Updated for Kinesis and removed Kafka as I recently learnt this isn't supported yet.